.cbmc-batch/jobs/aws_cryptosdk_keyring_trace_add_record/aws_cryptosdk_keyring_trace_add_record_harness.c

```diff
@@ -33,6 +33,9 @@ void aws_cryptosdk_keyring_trace_add_record_harness() {
         ensure_string_is_allocated_bounded_length(MAX_STRING_LEN); /* Precondition: name must be non-null */
     uint32_t flags;
 
+    __CPROVER_assume(namespace != NULL);
+    __CPROVER_assume(name != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated to do this and to match the precondition in the harness. I think originally I thought making the non-NULL assumptions as clear as possible was the goal which is why I used the !=NULL assumptions, but I prefer the is_valid ones so happy to just switch over to those.